### ISSUE TITLE
[C++] unable to parse move operators correctly

### ISSUE THREAD

**Issue by jasjuang** [20Tauri/DoxyDoxygen#122]
Suppose we have a function below

```
void abc(int &&a);
```

alt+q yields

```
/**
 * @brief      { function_description }
 *
 * @param[in]  <unnamed>  { parameter_description }
 */
void abc(int &&a);
```

but I am expecting 

```
/**
 * @brief      { function_description }
 *
 * @param      a     { parameter_description }
 */
void abc(int &&a);
```

This is minor inconvenience but still looking forward to get this fixed!

**Comment by 20Tauri** [comment]
Thanks for reporting,

It's a leak in C++11 support. The fix will be deployed next week on evolution branch

**Comment by 20Tauri** [comment]
Fixed in [0.68.3](https://github.com/20Tauri/DoxyDoxygen/releases/tag/0.68.3)
Available from now in evolution branch.
